test/plugin.html-validate.external-links.mjs

```diff
@@ -70,6 +69,7 @@ export default class ExternalLinksRule extends Rule {
         type: "string",
         description:
           "URL of proxy server to check external links. Must accept a query parameter 'url' and return the status code and possibly location: header in the response.",
+        required: true,
```

Comment:
this option is NOT required.

if it is unspecified then the proxy server is not used, as was the case before you started working on this branch.